### DISCUSSION TITLE
Sievert values

### DISCUSSION THREAD

**Discussion by AF1977** [OpenGammaProject/Open-Gamma-Detector#38] [Q&A] [answered]
Hi,

Is it possible to get Sievert values instead of cps?
As far as I know I have to look at the whole spectrum and weigh the individual parts to get an accurate value for Sievert.
Or not?

Thanks in advance.

BR

Andreas

**Comment by NuclearPhoenixx** [comment]
Hi again,

that's actually not trivial since for different sizes and geometries of scintillator crystals produce different count rates for the same sample.
Then you also have different software settings that might interfere with it too. And most importantly, it also depends on the on-board pots and how you set those (SiPM voltage, threshold, etc.).

If you have a calibrated geiger counter at hand, you could use this as a way to somewhat calibrate your device. The simplest possible way is to just find a multiplication factor that converts your cps reading to a Sv reading. You can play around with this factor until the Sv reading on your calibrated Geiger counter matches the one on your device. That's not ideal at all, but it's quick and easy.

A small disclaimer: I intentionally didn't add a Sievert reading. That's because it might lead you to believe that the device is a genuine measuring instrument for nuclear safety and you might want to use it for such a purpose. Do NOT do this. This is more of a toy project than something you want to rely on. Furthermore, using a spectrometer as a Geiger counter only, in order to read Sievert is quite the waste of resources. There's much cheaper projects suited for this use case, such as my [Mini SiD](https://github.com/OpenGammaProject/Mini-SiD) project.

For everything else, a simple cps reading is more than enough to gauge the activity relative to the background.

Hope this answers your question!

**Reply by AF1977** [reply]
Hi,

Thanks for the anwer. I don't want to use this for a quantitative measurement, but rather for a qualitative, orienting measurement. I work as a specialist for occupational safety. It is very often the case that a measurement first shows people, "Hey, something is not ok here.." And an orienting measurement is ok with your project. I have contacts with the professional fire brigade, so I would like to carry out comparative measurements. And your "toy" is very close to professional devices, believe me.

BR

Andreas

**Reply by NuclearPhoenixx** [reply, marked as answer]
You can try the simple multiplication way.

You will need to change this line here (this is the display for the geiger mode): https://github.com/OpenGammaProject/Open-Gamma-Detector/blob/f222fe1e3fdea683bb4137fc98f3553db76b0cbb/software/opengamma_pico/opengamma_pico.ino#L827

Try to find a constant that, when multiplied with `avg_cps`, will output Sv values close to a calibrated one in the same environment (e.g. background only). If your're in an order of magnitue of like 10 at background levels, you'd probably want to start with a factor of around 1/100 000 000 and test forward from there.

I don't know for sure if this works for larger ranges (linearity!), but you could also try to build a linear function that converts cps to Sv if you have two pairs of cps and Sv values from a known calibrated device. Ideally, these two pairs would be as far away as possible in terms of cps. There are a lot of polynomial solvers out there that will spit you out the correct linear function for a two pairs.
